components/input/index.en-US.md

```diff
@@ -107,6 +107,7 @@ The rest of the props of `Input.TextArea` are the same as the original [textarea
 | Property | Description | Type | Default |
 | --- | --- | --- | --- |
 | enterButton | Whether to show an enter button after input. This property conflicts with the `addonAfter` property | boolean \| ReactNode | false |
```

Comment:
That could also work but it i think it would be nice to be able to control the already made button.
Or instead of my feature, should i create a new proprety (like buttonProps) that could send props directly to the Button?